src/js/component.js

```diff
@@ -1023,6 +1041,7 @@ class Component {
     const report = Lib.bind(this.player(), this.player().reportUserActivity);
 
     let touchHolding;
```

Comment:
These floating lets feel weird to me. If it makes sense we might try to group them with other lets/consts.